App.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import Instant2FAComponent from './src/Instant2fa-component.jsx';
+import Instant2FAComponent from './src/Instant2FAComponent.jsx';
```

Comment:
Is this an example or should it be removed? If it's an example can you move it to an `example/` directory?